### ISSUE TITLE
Checklist design review for v22

### ISSUE THREAD

**Issue by lewispg228** [sparkfun/RedBoard#1]
1) We need to reconsider the resistor values on the LEDs. They are _very_ bright. This has been a common complaint from EDU, and there are lots of comments on the product page about this being an issue.

2) Depending on what the DFM agrees to this Friday, I'm suggesting that we revert the silk to the previous style for the 10uF caps (C3, C6 and C13). I think the line for the positive side of a polarized cap will be confusing. 

*I'll wait to finish BOM verification until we decide what to do here.

3) The mark on the STAT LED (13) is not ideal. I suggest we possibly revert these packages as well for this rev. Or we flip the LED 180 degrees.

![image](https://cloud.githubusercontent.com/assets/601236/16495908/a0bf363c-3eac-11e6-9e93-6385eedf55cc.png)

4) should probably get the "v22 revised by: jim" in the sch and brd too. Also, the board is missing the other "based on:...." text objects.

5) Do we want to put a line over RESET pin silk? (it isn't really standard for us, but it is an active low pin, right?)

**Comment by mgrusin** [comment]
We in Edu have noticed odd behavior when running higher-current loads such as motors through the board to VIN; things like the board browning out etc. We support reviewing and upgrading the power paths to make sure they're extra-wide, smooth, and well-isolated.

**Comment by jimblom** [comment]
@lewispg228 1. 4.7k on the power LED seems to cut the green LED's intensity down to a level similar to the blue pin 13 LED. That's the only one that seems really egregious to me.
2. Whatever's best for production. We can talk about it Friday.
3. Agreed. Reverted back to old silkscreen to avoid changing copper.
4. I added the "Based on..." text to the board. If I'm the original designer, is a "revised by me" necessary? Eh.
5. Noticed that too, but I'm trying to avoid dramatically changing any silk. There's precedent for the bar-less RESET label on other Arduino boards, I think it's safe to leave off this one.

Changes pushed.
